tests/consensus_tests/test_replicate_points.py

```
@@ -44,14 +46,18 @@ def test_replicate_points_stream_transfer(tmp_path: pathlib.Path):
     upsert_random_points(peer_api_uris[0], 100, shard_key="default")
 
     # Transfer shard from one node to another
-    filter = {}
+    filter = {"should": {"key": "city", "match": {"value": "London"}}}
+
+    expected_count = get_collection_point_count(peer_api_uris[0], COLLECTION_NAME, shard_key="default", exact=True, filter=filter)
```

Ruff (0.14.2)

`get_collection_point_count` may be undefined, or defined from star imports

(F405)